sklearn/semi_supervised/_label_propagation.py

```diff
@@ -157,11 +157,11 @@ def predict(self, X):
 
         Parameters
         ----------
-        X : array-like of shape (n_samples, n_features)
+        X : ndarray of shape (n_samples, n_features)
```

Comment:
For parameters to `fit`, `predict` and a few others, we should keep `array-like`, which is more general than `ndarray`

You can keep `ndarray` for return types

Reply:
Thanks @NicolasHug for reviewing the PR,Have made the edits for keeping array-like everywhere other than return types